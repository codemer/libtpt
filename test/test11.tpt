@set(result, !0)\
!0=${result}
@set(result, !1)\
!1=${result}
@set(result, 10 == 10)\
10==10=${result}
@set(result, 10 != 10)\
10!=10=${result}
@set(result, 10 < 100)\
10<100=${result}
@set(result, 10 > 100)\
10>100=${result}
@set(result, 10 <= 100)\
10<=100=${result}

