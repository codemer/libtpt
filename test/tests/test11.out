!0=1
!1=0
10==10=1
10!=10=0
10<100=1
10>100=0
10<=100=1

